src/commands/createHttpFunction.ts

```diff
@@ -22,7 +23,14 @@ export async function createHttpFunction(context: IActionContext): Promise<void>
     const funcApi: AzureFunctionsExtensionApi = await getFunctionsApi(context);
 
     const endpointName: string = 'endpoint';
-    const folderPath: string = path.join(vscode.workspace.workspaceFolders[0].uri.fsPath, defaultApiName);
+    const apiLocationPlaceholder: string = getWorkspaceSetting(apiSubpathSetting, vscode.workspace.workspaceFolders[0].uri.fsPath) || defaultApiName;
```

Comment:
> I think we also want to prompt the user for a folder name (and have the default value be api or the workspaceSetting)

The purpose of the setting is to avoid a prompt, not to be the default value for a prompt. Also - I never prompt for a folder before creating a function in the functions extension. I think this experience needs to be ironed out a bit more so that it's not prompting too much

Reply:
Okay then let's roll back to Alex's last commit to where it just replaces 'api' with the workspace setting.  That should at least unblock some users

Reply:
Rolled back ✔️